src/drawer.canvasentry.js

```diff
@@ -103,11 +104,11 @@ export default class CanvasEntry {
         // decimal between 0 and 1
         this.start = this.wave.offsetLeft / totalWidth || 0;
         this.end = this.start + elementWidth / totalWidth;
+        const elementSize = { width: `${elementWidth}px` };
```

Comment:
I'm gonna leave this change in because I prefer template literals to string concatenation.